TypeCobol/Compiler/Symbols/Symbol.cs

```diff
@@ -76,6 +77,8 @@ public enum Flags : ulong
             InsideTypedef = 0x01L << 35,//Flag of any symbol inside a Typedef definition.
             BuiltinSymbol = 0x01L << 36, //This is a Builtin symbol.
             
+            Declaratives = 0x01L << 37,
```

Comment:
Actually, this flag is only set, never tested for existence. So it can be removed.

Reply:
No the initial purpose was to flag BasicBlock entry  of Declaratives, I thought this was done, but It seems that I forgot to do that. So one could check if the instruction of the block is declarative, but I think it's better to flag such blocks. So it still to be done.

Reply:
See PR #1738, Symbol.cs reverted to its state in develop branch.